packages/phone-number-privacy/src/common/identity.ts

```diff
@@ -8,3 +11,14 @@ export function authenticateUser() {
   // TODO [amyslawson]
   return
 }
+
+export async function isVerified(account: string, hashedPhoneNumber: string): Promise<boolean> {
+  // TODO (amyslawson) wrap forno request in retry
+  // TODO (aslawson) update to work with hashed phoneNumber
+  const attestationsWrapper: AttestationsWrapper = await getContractKit().contracts.getAttestations()
+  const attestationStats = await attestationsWrapper.getAttestationStat(hashedPhoneNumber, account)
+  const numAttestationsCompleted = attestationStats.completed
+  const numAttestationsRemaining =
+    config.attestations.numberAttestationsRequired - numAttestationsCompleted
```

Comment:
Broader question: should we also take into account success rate (in addition to min attestations). For example, if I control a validator node, I may be able to fake 1/100 attestations so I could reach the min attestation threshold but my success rate would be 1%.

Reply:
technically yes -- the `getAttestationStat()` also returns `total` attestations.

my worry is what if there are user issues -- SMS get lost, the user gets confused and then resubmits for new attestation requests?  What would be the appropriate percentage to balance affect on user versus potential malicious request?

also, considering the number of require attestations is currently 3, you'd need to run 3 nodes that all get randomly selected to perform the attestation to hack this.

for reference, this is how the wallet confirms a phone number is verified currently.  though I'm open to the suggestion.

Reply:
@codyborn We definitely should take into account success rate and we intend to do the same in the wallet. The exact tuning of that isVerified is still TBD but we'll probably want to move that logic to ContractKit when we have it

Reply:
@jmrossy do you suggest we should add success rate now?  and what do we think would be a good starting point for the threshold?

Reply:
Makes sense. We can take a look at the success rates for non-malicious users and base it off of that.